stubs/tqdm/tqdm/asyncio.pyi

```diff
@@ -84,7 +84,7 @@ class tqdm_asyncio(std_tqdm[_T]):
     @overload
     def __init__(
         self,
-        iterable: Iterable[_T],
+        iterable: Iterable[_T] | Iterator[_T] | AsyncIterator[_T],
```

Comment:
`Iterable` is redundant, as each `Iterator` is an `Iterable`.

```suggestion
        iterable: Iterator[_T] | AsyncIterator[_T],
```

Reply:
`Iterable[_T]` is still a valid type, so `Iterator[_T]` should probably be removed instead.

Reply:
Looks to me that @mattmess1221  is correct, just because this code fallsback to iter(iterable), so they will create an Iterator themselves from Iterable.
The code in question: https://github.com/tqdm/tqdm/blob/0ed5d7f18fa3153834cbac0aa57e8092b217cc16/tqdm/asyncio.py#L26-L34

Reply:
You are correct, of course. `Iterator` is redundant.